### PR TITLE
build: refactor/building

### PR DESCRIPTION
**Pull request by jacoobes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "@sern/handler",
     "packageManager": "pnpm@7.28.0",
     "version": "2.6.0",
-    "description": "A customizable, batteries-included, powerful discord.js framework to automate and streamline bot development.",
+    "description": "A complete, customizable, typesafe, & reactive framework for discord bots.",
```
Comment (Murtatrxx) [inline package.json:5]: I don't think we should say "for discord bots"

File: README.md [hunk 2]
```diff
@@ -117,4 +109,18 @@ It is **highly encouraged** to use the [command line interface](https://github.c
...
+```
+- Copy
+```sh
+git clone https://github.com/sern-handler/handler.git
+```
+- Create bundle
+```sh
+//prod for production, dev for dev build
+pnpm run build:(prod|dev)
+```
```
Comment (Murtatrxx) [inline README.md:126]: Do you really think people will primarily focus on these? We can also include these in docs

**Review by Murtatrxx — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Murtatrxx) [reply package.json:5]: To keep it simple, and short.

File: README.md (hunk 2 above)
Comment (jacoobes) [reply README.md:126]: Its for those who want to contribute and quick-start build. Do you think we should move this somewhere else?

File: package.json (hunk 1 above)
Comment (jacoobes) [reply package.json:5]: Any alternatives?

File: README.md (hunk 2 above)
Comment (Murtatrxx) [reply README.md:126]: Yeah it'll fit better, we may lose some users due to a long README section

File: package.json (hunk 1 above)
Comment (Murtatrxx) [reply package.json:5]: It's fine, just remove "for discord bots."

File: README.md (hunk 2 above)
Comment (jacoobes) [reply README.md:126]: Ok. I was thinking I can link how to build, which will redirect to the website. how does that sound?

File: package.json (hunk 1 above)
Comment (Murtatrxx) [reply package.json:5]: its more up to you, and how you like it

File: README.md (hunk 2 above)
Comment (Murtatrxx) [reply README.md:126]: That works, yeah.

**Review by Murtatrxx — APPROVED** [review]

**Pull request merged**